### PR TITLE
Feat/#125

### PR DESCRIPTION
**Pull request by Dawid-Dwk**

### CODE REVIEW

File: packages/backend/src/modules/dashboard/column.mapper.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import ColumnDTO from 'packages/shared/lib/api/dto/column.dto'
+import { Mapper } from '../../application/Mapper'
+import { Column } from './column'
+
+export class ColumnMapper implements Mapper<Column, any, ColumnDTO> {
+  public mapToDomain(raw: any): Column {
+    return new Column(raw.idCol, raw.title, raw.order, raw.tasks)
```
Comment (mcwiekala) [inline packages/backend/src/modules/dashboard/column.mapper.ts:7]: change idCol to id

Comment (mcwiekala) [reply packages/backend/src/modules/dashboard/column.mapper.ts:7]: for tasks use TaskMapper

File: packages/backend/src/modules/dashboard/column.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import TaskBase from '../task/task'
+
+export class Column {
+  private _idCol: string
```
Comment (mcwiekala) [inline packages/backend/src/modules/dashboard/column.ts:4]: change to: id

File: packages/backend/src/modules/dashboard/column.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  private _title: string
+  private _order: number
+  private _tasks: TaskBase[]
```
Comment (mcwiekala) [inline packages/backend/src/modules/dashboard/column.ts:7]: don't use TaskBase, use just Task or TaskDTO

File: packages/backend/src/modules/dashboard/dashboard.mapper.ts [hunk 4]
```diff
@@ -5,7 +5,15 @@ import { Mapper } from '../../application/Mapper'
...
   public mapToDomain(raw: any): Dashboard {
-    return new Dashboard(raw._id, raw.title, raw.description, raw.imageCoverUrl, raw.status)
+    return new Dashboard(
+      raw._id,
+      raw.title,
+      raw.description,
+      raw.imageCoverUrl,
+      raw.createdAt,
+      raw.status,
+      raw.column
```
Comment (mcwiekala) [inline packages/backend/src/modules/dashboard/dashboard.mapper.ts:15]: Does it work? here you use ColumnDTO to app to domain type. It should be mapper from ColumnDTO to Column

File: packages/backend/src/modules/dashboard/dashboard.model.ts [hunk 5]
```diff
@@ -4,8 +4,9 @@ import { DashboardVisibility } from 'shared'
 const { ObjectId } = mongoose.Types
 
 const columnSchema = new mongoose.Schema({
-  _id: ObjectId,
```
Comment (mcwiekala) [inline packages/backend/src/modules/dashboard/dashboard.model.ts:7]: why this change?

File: packages/backend/src/modules/dashboard/dashboard.controller.ts [hunk 6]
```diff
@@ -39,6 +39,26 @@ class DashboardController {
       return res.status(400).send()
     }
   }
+
+  async removeDashboardById(req: express.Request, res: express.Response) {
+    const dashboard: Dashboard | null = await this.dashboardService.removeDashboardById(
```
[the remark this answers is not in the document]
Comment (mcwiekala) [reply packages/backend/src/modules/dashboard/dashboard.controller.ts:44]: use try-catch to handle errors like here:
```
  async findById(req: express.Request, res: express.Response) {
    try {
      const task: Task = await this._taskService.findById(req.params.taskId)
      const taskDto: TaskDTO = this._taskMapper.mapToDto(task)
      return res.status(201).send(taskDto)
    } catch (error) {
      if (error instanceof CannotFindDocumentError) {
        return res.status(404).send('Task not found')
      }
      return res.status(400).send('Bad Request')
    }
  }
```

File: packages/backend/src/modules/dashboard/column.ts (hunk 3 above)
Comment (Dawid-Dwk) [reply packages/backend/src/modules/dashboard/column.ts:7]: done

File: packages/backend/src/modules/dashboard/dashboard.model.ts (hunk 5 above)
Comment (Dawid-Dwk) [reply packages/backend/src/modules/dashboard/dashboard.model.ts:7]: with "_id" mongo doesn't generate id, if you have another solution for this problem I'm open to it
